### DISCUSSION TITLE
Is there a way to load a specific node of VirtualTourPlugin??

### DISCUSSION THREAD

**Discussion by egpgit** [mistic100/Photo-Sphere-Viewer#1489] [General]
Hi everybody!

I am a newbie and I have made an html with a 10 nodes tour using de VirtualTourPlugin.
With "startNodeId" I setted the default start node, but I would like to change "startNodeId" depending on the website from which I link to the tour.

This is the case:
The Tour shows a building, now I only have external Tour made, but in a future I will make internal tour of the building.
The building has several entrances

I would like to be able to access the external tour from the node of the North gate or from the node of the South gate, so when loading the page of the external tour, the initial node would be 1 if I have accessed from the north gate or 2 if I have accessed from the south.

The way I had in mind to link the 2 tours is through a marker and a listener to said marker so that when the user clicks I make a document.location to the web of the external tour

Is it possible? is there any alternative?

Thanks a lot!

**Comment by mistic100** [comment]
https://letmegooglethat.com/?q=javascript+get+query+param

```js
startNodeId: new URLSearchParams(window.location.search).get('startId')
```

And then use https://website.com/tour.html?startId=4

**Reply by egpgit** [reply]
Thanks so much!!
I'll try it
